### PR TITLE
Highlight IC events

### PR DESCRIPTION
**Pull request by Remg**
Closes [#667](https://app.clubhouse.io/enmarche/story/667/ic-page-d-affichage-des-%C3%A9v%C3%A9nements-donner-une-couleur-diff%C3%A9rente-aux-ic-pour-les-distinguer-des-%C3%A9v%C3%A9nements-d-al)

### CODE REVIEW

File: templates/adherent/events.html.twig [hunk 1]
```diff
@@ -54,7 +54,11 @@
                                     {{ event.beginAt|localizeddate('none', 'none', 'fr_FR', null, 'd MMMM Y') }} &mdash; à {{ event.cityName }}<br />
                                 </div>
                                 {% if event.type and event.isCitizenInitiative %}
-                                    <div class="text--body"><span class="text--bold">Initiative citoyenne</span> organisée par l'adhérent <span class="text--bold">{{ event.organizer.fullName|default('Anonyme') }}</span></div>
+                                    {% if event.organizer %}
```
Comment (illusionOfParadise) [inline templates/adherent/events.html.twig]: No need of `if` and even of `|default('Anonyme')`

Comment (Remg) [reply templates/adherent/events.html.twig]: Shouldn't we keep consistency with the event list of the search template ?
https://github.com/EnMarche/en-marche.fr/blob/master/templates/search/_events.html.twig#L27

or should we refactor both ?

Comment (Remg) [reply templates/adherent/events.html.twig]: got it. this check is useless on this page

Comment (illusionOfParadise) [reply templates/adherent/events.html.twig]: Sorry for this erroneous comment. 
This page lists events you are subscribed.
Your code is good !

**Pull request merged**